Design System Deployment development to staging

VenuKG opened the pull request.

alok-nakhate reviewed (approved):
Changes are reviewed and approved.

nitinsonawane123 committed:
Commit Changes for task 849

nitinsonawane123 committed:
Commit Changes for bug 1322

nitinsonawane123 committed:
Commit Changes for bug 1361

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1361-elements-file-uploader-is-not-display-proper

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1322-components-email-settings-label-input-box-alignment-issue

nitinsonawane123 committed:
Commit Changes for bug 1322

nitinsonawane123 committed:
Commit Changes for bug 1354

nitinsonawane123 committed:
Commit Changes for bug 1347

SoheshMG committed:
updated the value array and changed position of  previous buttons

nitinsonawane123 committed:
Commit Changes for bug 1353

SoheshMG committed:
updated the maxRecordsPage condition

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/task-849-element-progress-bar-enhancements-in-style

nitinsonawane123 committed:
Commit Changes for bug 1348

nitinsonawane123 committed:
Commit Changes for bug 1360

nknikhilkasar committed:
Add advance stepper functionality with customizable properties

SoheshMG committed:
Reopen Feed changes added size property

SoheshMG committed:
Reopen adjusted buttons in Premium Support

SoheshMG committed:
Reopen personal info button alignment

nitinsonawane123 committed:
Commit Changes for task 784

Dipak7028 committed:
updated rds comp grid tsx file

Dipak7028 committed:
Merge branch 'development' into users/dipak/bug-1404-design-system-components-grid-overlapping-issue

nknikhilkasar committed:
Merge pull request #1413 from Wai-Technologies/user/nikhil/enhance-1387-add-advance-stepper

Enhanced 1387: Design System > Elements > Stepper > Enhancement in stepper

nitinsonawane123 committed:
Commit Changes for bug 1412

Sachin-Ghorpade committed:
Committed changes on banner light theme text is not display

nknikhilkasar committed:
Enhance password validation logic and add confirmation checks in password settings component

nknikhilkasar committed:
Merge branch 'development' into users/dipak/bug-1404-design-system-components-grid-overlapping-issue

Dipak7028 committed:
Merge pull request #1415 from Wai-Technologies/users/dipak/bug-1404-design-system-components-grid-overlapping-issue

Resolved Bug 1404 :  Design System > Components > Grid > Overlapping issue

nitinsonawane123 committed:
Merge branch 'development' into users/sachin/bug-1282-design-system-baner-light-theme-text-is-not-display

Sachin-Ghorpade committed:
Merge pull request #1419 from Wai-Technologies/users/sachin/bug-1282-design-system-baner-light-theme-text-is-not-display

Resolved Bug 1282: Design System > Baner > Raaghu > light theme text is not display

nitinsonawane123 committed:
Merge branch 'development' into users/nikhil/bug-1368-password-validation-for-current-password-and-new-password

nknikhilkasar committed:
bug 1368 changed variable name

Sachin-Ghorpade committed:
Committed changes on  Cover Image  Blog Id input box spacing issue

nknikhilkasar committed:
Merge pull request #1420 from Wai-Technologies/users/nikhil/bug-1368-password-validation-for-current-password-and-new-password

Resolved Bug 1368 : Design System > Components > Password Setting > Default > The new password and confirm new password validation are not displayed properly.

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1348-component-personal-info-verify-email-button-alignment-issue

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1311-design-system-elements-pagination-advanced-dark-theme-pagination-is-not-display-proper

nknikhilkasar committed:
Merge branch 'development' into users/nitin/task-849-element-progress-bar-enhancements-in-style

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1360-elemets-fab-menu-small-mobile-view-is-not-display-proper

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1412-element-dropdown-multiselect-selected-option-is-not-display-proper

SoheshMG committed:
Merge pull request #1423 from Wai-Technologies/users/sohesh/bug-1311-design-system-elements-pagination-advanced-dark-theme-pagination-is-not-display-proper

Resolved Bug 1311 : Design System > Elements > Pagination > Advanced > Dark theme pagination is now display proper

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1347-components-page-not-found-text-is-not-displaying-center

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1348-component-personal-info-verify-email-button-alignment-issue

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1360-elemets-fab-menu-small-mobile-view-is-not-display-proper

nitinsonawane123 committed:
Commit Changes for bug 1348

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1361-elements-file-uploader-is-not-display-proper

nitinsonawane123 committed:
resolved conflict

nknikhilkasar committed:
Merge branch 'development' into user/nitin/task-784-elements-enhancement-in-action-button-dropdown

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1354-components-visual-setting-after-submitting-select-list-values-are-not-clear

Dipak7028 committed:
Resolve issues in Elements >Notification > Default > Mark As Read button alignment issue.

Resolved issue by  Mark As Read button alignment issue in changing by m-3 to m-2

nknikhilkasar committed:
Merge branch 'development' into users/sachin/bug-1340-design-system-components-blog-post-default-cover-image-blog-id-input-box-spacing-issue

Dipak7028 committed:
Merge pull request #1425 from Wai-Technologies/users/dipak/1395-design-system-elements-time-picker-default-outside-icon-click-functionality-is-not-working

Resolved bug 1395: Elements >Notification > Default > Mark As Read button alignment issue.

nitinsonawane123 committed:
Merge pull request #1408 from Wai-Technologies/users/nitin/bug-1360-elemets-fab-menu-small-mobile-view-is-not-display-proper

Resolved bug 1360 : Design System > Elements > Fab Menu > Small mobile view is not display proper.

nitinsonawane123 committed:
Merge pull request #1406 from Wai-Technologies/users/nitin/bug-1348-component-personal-info-verify-email-button-alignment-issue

Resolved bug 1348 : Design System > Components > Personal Info > Default > Verify Email button alignment issue.

nitinsonawane123 committed:
Merge pull request #1400 from Wai-Technologies/users/nitin/bug-1347-components-page-not-found-text-is-not-displaying-center

Resolved bug 1347 : Design System > Components > Page not found > Default > Text is not displaying center.

nitinsonawane123 committed:
Merge pull request #1325 from Wai-Technologies/users/nitin/bug-1322-components-email-settings-label-input-box-alignment-issue

Resolved bug 1322 : Design System > Components > Email Settings > Default > Label & Input box alignment issue.

Sachin-Ghorpade committed:
Merge branch 'development' into users/sachin/bug-1340-design-system-components-blog-post-default-cover-image-blog-id-input-box-spacing-issue

Sachin-Ghorpade committed:
Merge pull request #1421 from Wai-Technologies/users/sachin/bug-1340-design-system-components-blog-post-default-cover-image-blog-id-input-box-spacing-issue

Resolved Bug 1340: Design System > Components > Blog Post > Default > Cover Image & Blog Id input box spacing issue.

nitinsonawane123 committed:
Commit Changes for bug 1353

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1412-element-dropdown-multiselect-selected-option-is-not-display-proper

Dipak7028 committed:
Resolved Bug 1404 : Design System > Elements > Rating > The position of the rating stars is inconsistent after changing the count

Resolve issues in  Elements > Rating > The position of the rating stars is inconsistent after changing the count

Resolved issue by add a justify-content-between and remove the pe-5 in class

nitinsonawane123 committed:
Commit Changes for bug 1412

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1354-components-visual-setting-after-submitting-select-list-values-are-not-clear

rameshwarkamandar committed:
Add missing controls, update interface

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/task-849-element-progress-bar-enhancements-in-style

nitinsonawane123 committed:
Merge pull request #1300 from Wai-Technologies/users/nitin/task-849-element-progress-bar-enhancements-in-style

Completed Task 849 : Design System > Element > Progress Bar > Default : Enhancements in style.

Dipak7028 committed:
Merge branch 'development' into users/dipak/bug-1393-design-system-Elements-Rating-the-position-of-the-rating-stars-is-inconsistent-after-changing-the-count

nknikhilkasar committed:
Merge branch 'development' into users/rameshwar/bug/1427-elements-app-detail-Missing-controls-and-missing-formatting

supriyartambewaiin committed:
Merge branch 'development' into users/nitin/bug-1412-element-dropdown-multiselect-selected-option-is-not-display-proper

Dipak7028 committed:
Merge pull request #1426 from Wai-Technologies/users/dipak/bug-1393-design-system-Elements-Rating-the-position-of-the-rating-stars-is-inconsistent-after-changing-the-count

Resolved Bug 1404 : Design System > Elements > Rating > The position of the rating stars is inconsistent after changing the count

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1242-add-parameter-size-for-avatar-and-updating-other-elements

nitinsonawane123 committed:
Merge pull request #1418 from Wai-Technologies/users/nitin/bug-1412-element-dropdown-multiselect-selected-option-is-not-display-proper

Resolved bug 1412 : Design System > Elements > Dropdown > With Multi Select > Small & large mobile selected option is not display proper.

SoheshMG committed:
changes in rds-select-list.css

nitinsonawane123 committed:
Commit Changes for bug 1354

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1354-components-visual-setting-after-submitting-select-list-values-are-not-clear

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1361-elements-file-uploader-is-not-display-proper

nitinsonawane123 committed:
Merge pull request #1367 from Wai-Technologies/users/nitin/bug-1361-elements-file-uploader-is-not-display-proper

Resolved bug 1361 : Design System > Elements > File Uploader > Default > File Uploader is not display proper.

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1344-claims-default-input-box-highlighted-in-different-color

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

supriyartambewaiin committed:
Merge branch 'development' into users/nitin/bug-1354-components-visual-setting-after-submitting-select-list-values-are-not-clear

nitinsonawane123 committed:
Merge pull request #1398 from Wai-Technologies/users/nitin/bug-1354-components-visual-setting-after-submitting-select-list-values-are-not-clear

Resolved bug 1354 : Design System > Components > Visual Setting > Default > After submitting select list values are not clear.

rameshwarkamandar committed:
Added sample link url

nknikhilkasar committed:
Merge branch 'development' into users/rameshwar/bug/1427-elements-app-detail-Missing-controls-and-missing-formatting

rameshwarkamandar committed:
Merge pull request #1428 from Wai-Technologies/users/rameshwar/bug/1427-elements-app-detail-Missing-controls-and-missing-formatting

Resolved Bug 1427: Design System > Element > App detail > Default > Missing Controls and missing formatting.

nitinsonawane123 committed:
Commit Changes for bug 1353

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

Sachin-Ghorpade committed:
Committed changes on checkbox and text alignment issue and validation pattern

nitinsonawane123 committed:
Merge pull request #1405 from Wai-Technologies/users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

Resolved bug 1353 : Design System > Components > Tenant Information > Default > After submitting field not clear.

Sachin-Ghorpade committed:
Committed changes on Validation message should be same as per other components

Sachin-Ghorpade committed:
Committed changes on validation msg

supriyartambewaiin committed:
Merge branch 'development' into users/sachin/bug-1346-design-system-components-database-connection-checkbox-and-text-alignment-issue

Sachin-Ghorpade committed:
Merge pull request #1433 from Wai-Technologies/users/sachin/bug-1346-design-system-components-database-connection-checkbox-and-text-alignment-issue

Resolved Bug 1346: Design System > Components >Database Connection > checkbox and text alignment issue and Database URL validation

SoheshMG committed:
Changed RdsTextArea to RdsTextEditor

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1242-add-parameter-size-for-avatar-and-updating-other-elements

SoheshMG committed:
Merge pull request #1429 from Wai-Technologies/users/sohesh/bug-1242-add-parameter-size-for-avatar-and-updating-other-elements

Resolved Bug 1242 : Design system > Elements > Feed > Added size parameter and Resolved Reopen Bugs

supriyartambewaiin committed:
Merge branch 'development' into users/sohesh/bug-1344-claims-default-input-box-highlighted-in-different-color

rameshwarkamandar committed:
Resolved bug: 1434 -Code formatting and optimization.

Dipak7028 committed:
Resolved Bug 1293: Design system > Components > Edition Information > Small large tablet button alignment

1395 Resolve issues in design ystem components edition information small large tablet button alignment issue.

nitinsonawane123 committed:
Merge branch 'development' into user/nitin/task-784-elements-enhancement-in-action-button-dropdown

Dipak7028 committed:
Resolved Bug 1293: Design system > Components > Edition Information >Small large tablet button alignment

1395 Resolve issues in design ystem components edition information small l arge tablet button alignment issue.

-Resolved issue by Mark As small large tablet button alignment issue alignment issue in changed
Edition Information > Small, Large, Tablet button alignment should be proper.
Email Setting >Default > All view alignment should be proper.

Screenshot

Dipak7028 committed:
Merge pull request #1436 from Wai-Technologies/users/dipak/1293-design-system-components-edition-information-small-large-tablet-button-alignment-issue

Resolved Bug 1293: Design system > Components > Edition Information >Small large tablet button alignment

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1344-claims-default-input-box-highlighted-in-different-color

SoheshMG committed:
Merge pull request #1431 from Wai-Technologies/users/sohesh/bug-1344-claims-default-input-box-highlighted-in-different-color

Resolved Bug 1344 : Design System > Components > Claims And Developer Mode > Default > Input box highlighted in required color Reopen Bug

rameshwarkamandar committed:
Merge branch 'development' into user/rameshwar/bug/1434-elements-alerts-Code-formatting-and-optimization

rameshwarkamandar committed:
Merge pull request #1435 from Wai-Technologies/user/rameshwar/bug/1434-elements-alerts-Code-formatting-and-optimization

Bug 1434: Resolved Design System > Elements > Alerts > Code formatting and optimization

nitinsonawane123 committed:
Commit Changes for task 784

supriyartambewaiin committed:
Merge branch 'development' into user/nitin/task-784-elements-enhancement-in-action-button-dropdown

nitinsonawane123 committed:
Merge pull request #1414 from Wai-Technologies/user/nitin/task-784-elements-enhancement-in-action-button-dropdown

Completed Task 784 : Design System > Elements > Enhancements in Action Button/Dropdown.

rameshwarkamandar committed:
Fixed deployment issue

rameshwarkamandar committed:
Merge pull request #1437 from Wai-Technologies/users/rameshwar/fixed-deployment-issue

Fixed deployment issue

Dipak7028 committed:
Resolved Bug 1393: Design System > Elements > Rating > The position of the rating stars is inconsistent after changing the count

Dipak7028 committed:
Merge pull request #1438 from Wai-Technologies/users/dipak/bug-1393-design-system-Elements-Rating-the-position-of-the-rating-stars-is-inconsistent-after-changing

Resolved Bug 1393: Design System > Elements > Rating > The position of the rating stars is inconsistent after changing the count

VenuKG committed:
Merge branch 'development' into users/venu/design-system-deployment-dev-to-stage

supriyartambewaiin reviewed (approved):
PR Reviewed and Approved
